docs/core/deploying/index.md

```diff
@@ -39,86 +39,11 @@ There are also a few disadvantages:
 
 - It is possible for the .NET Core runtime and libraries to change without your knowledge in future releases. In rare cases, this may change the behavior of your app.
```

Comment:
* "It is" :point_right: "It's"
* Vaguely feels like an example is needed to flesh out the last remark "in rare cases ..."